samples/client/petstore/java/feign/src/main/java/org/openapitools/client/ApiClient.java

```diff
@@ -69,8 +69,6 @@ public ApiClient(String[] authNames) {
         auth = new HttpBasicAuth();
       } else if ("bearer_test".equals(authName)) {
         auth = new HttpBearerAuth("bearer");
-      } else if ("http_signature_test".equals(authName)) {
-        auth = new HttpBearerAuth("signature");
```

Comment:
note for discussion: when http_signature_test is kept, the samples contain this incorrect line of code